specs/online_order_spec.rb

```diff
@@ -3,88 +3,114 @@
 require 'minitest/skip_dsl'
 
 # TODO: uncomment the next line once you start wave 3
-# require_relative '../lib/online_order'
+require_relative '../lib/online_order'
 # You may also need to require other classes here
 
 # Because an OnlineOrder is a kind of Order, and we've
 # already tested a bunch of functionality on Order,
 # we effectively get all that testing for free! Here we'll
 # only test things that are different.
 
-xdescribe "OnlineOrder" do
+describe "OnlineOrder" do
   describe "#initialize" do
     it "Is a kind of Order" do
-      # Check that an OnlineOrder is in fact a kind of Order
+      all_orders = Grocery::OnlineOrder.all
 
-      # Instatiate your OnlineOrder here
-      # online_order =
-      # online_order.must_be_kind_of Grocery::Order
-    end
+      all_orders.class.must_equal Array
 
-    it "Can access Customer object" do
-      # TODO: Your test code here!
-    end
+      all_orders.each do |order|
+        order.must_be_instance_of Grocery::OnlineOrder
+      end
 
-    it "Can access the online order status" do
-      # TODO: Your test code here!
+      all_orders.count.must_equal 100
     end
   end
 
-  describe "#total" do
-    it "Adds a shipping fee" do
-      # TODO: Your test code here!
-    end
+  xit "Can access Customer object" do
+    # TODO: I did not create a Customer class.
+  end
+
+  it "Can access the online order status" do
+
+    all_orders = Grocery::OnlineOrder.all
+
+    all_orders[0][3].must_be_kind_of Symbol
```

Comment:
This isn't a 2D array, instead it's an array of OnlineOrders.  So this should be:
```ruby
all_orders[0].fulfillment_status.must_be_kind_of Symbol
```